app/src/util/tools.ts

```diff
@@ -44,16 +44,16 @@ export const mulBN = (a: BigNumber, b: number, scale = 10000): BigNumber => {
  */
 export const calcPrice = (holdingsBN: BigNumber[]): number[] => {
   const holdings = holdingsBN.map(h => new Big(h.toString()))
+
+  const doesHoldingsAreZero = holdings.every(h => h.toString() === '0')
```

Comment:
`hasZeroHoldings` or anything else with proper grammar.

Reply:
I will fix this...